pkg/components/enterprise.go

```diff
@@ -186,5 +191,6 @@ var (
 		ComponentTigeraCNI,
 		ComponentCloudControllers,
 		ComponentElasticsearchMetrics,
+		ComponentEsGateway,
```

Comment:
Minor: I think for acronyms the Go convention is to use all caps. So `ComponentESGateway`.